src/ElectricPokemon.java

```diff
@@ -0,0 +1,112 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon {
+
+    public static final String type = "electric";
+    private static final List<String> attacks = Arrays.asList("ThunderPunch", "ElectroBall", "Thunder", "VoltTackle");
```

Comment:
Je schrijft deze aanvallen met hoofdletters, maar in PokemonGymImpl wordt gecheckt op aanvallen met kleine letters. Daar gaat er ook een `.toLowerCase()` overheen, dus er is geen probleem, maar het is wel iets om bij stil te staan.